cloudferry/lib/os/image/glance_image.py

```diff
@@ -293,12 +293,14 @@ def get_ref_image(self, image_id):
                 glance_exceptions.NotFound,
                 glance_exceptions.HTTPInternalServerError,
                 ssl.ZeroReturnError,
+                glance_exceptions.HTTPException,
```

Comment:
Remove `glance_exceptions.NotFound` and `glance_exceptions.HTTPInternalServerError`, `HTTPException` is a base class to them.